human-forecasts/update-submissions.R

```diff
@@ -0,0 +1,278 @@
+library(googledrive)
+library(googlesheets4)
+library(dplyr)
+library(purrr)
+
+
+# Google sheets authentification -----------------------------------------------
+options(gargle_oauth_cache = ".secrets")
+drive_auth(cache = ".secrets", email = "epiforecasts@gmail.com")
+gs4_auth(token = drive_token())
+
+spread_sheet <- "1nOy3BfHoIKCHD4dfOtJaz4QMxbuhmEvsWzsrSMx_grI"
+identification_sheet <- "1GJ5BNcN1UfAlZSkYwgr1-AxgsVA2wtwQ9bRwZ64ZXRQ"
+
+
+# setup ------------------------------------------------------------------------
+# - 1 as this is usually updated on a Tuesday
+submission_date <- Sys.Date() - 1
```

Comment:
Change to Monday detector used elsewhere (feel free to add a utility like `last_monday`)